go-controller/pkg/ovn/controller/services/repair.go

```diff
@@ -129,6 +128,38 @@ func (r *repair) runBeforeSync() {
 	}
 }
 
+func getLBs(nbClient libovsdbclient.Client) ([]*ovnlb.LB, error) {
```

Comment:
this is code duplication with what you added in the previous commit for getServiceLBs isn't it? Can you change that function to call this function as well?

Reply:
hm... definitely! will do.